### ISSUE TITLE
Impossible to amend without editing a commit message

### ISSUE THREAD

**Issue by andrewshadura** [andrewshadura/git-crecord#4]
When amending a commit and not editing the message (because it's fine), an error occurs:
```
Aborting commit; you did not edit the message.
abort: commit failed: git exited with status 1
```

**Comment by pklapperich** [comment]
This is caused by using the `-t` template option. In main.py, rather than 

```
            if opts['message'] is None:
                util.system(['git', 'commit', '--no-status', '-t', name] + args + ['--'] + list(files),
                           onerr=util.Abort, errprefix=_("commit failed"))
            else:
                util.system(['git', 'commit', '--no-status', '-F', name] + args + ['--'] + list(files),
                           onerr=util.Abort, errprefix=_("commit failed"))
```

it can be changed to 

```
            if opts['message'] is None:
                util.system(['git', 'commit'] + args + ['--'] + list(files),
                           onerr=util.Abort, errprefix=_("commit failed"))
            else:
                util.system(['git', 'commit', '--no-status', '-F', name] + args + ['--'] + list(files),
                           onerr=util.Abort, errprefix=_("commit failed"))
```

which just causes git to use the default template and then amend works as expected. The default template contains more stuff (like untracked and unstaged files) but it seems fine generally.

**Comment by andrewshadura** [comment]
Fixed in 0a820c9, but not exactly the way I would like to.
